cs_tools/sync/postgres/syncer.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import annotations
+
+import pydantic
+import sqlalchemy as sa
+import sqlmodel
+from cs_tools.sync.base import DatabaseSyncer
+from typing import TYPE_CHECKING, Any, Literal ,Optional
+from pydantic_core import PydanticCustomError
+import pathlib
+import logging
+from sqlalchemy.orm import sessionmaker
+from cs_tools import __version__
+from cs_tools.sync import utils as sync_utils
+from cs_tools.sync.base import DatabaseSyncer
+
+# if TYPE_CHECKING:
+from cs_tools.sync.types import TableRows
+
+log = logging.getLogger(__name__)
+
+
+class Postgres(DatabaseSyncer):
+    """
+    Interact with Postgres Database
+    """
+    
+    __manifest_path__ = pathlib.Path(__file__).parent / "MANIFEST.json"
+    __syncer_name__ = "postgres"
+   
+    dbname: str 
+    user:  str 
+    password:  str 
+    host:  str 
+    port: int 
+    schema_ : str = pydantic.Field(default="PUBLIC",alias="schema")
+    authentication: Literal["basic", "key-pair", "sso", "oauth"]
+    log_level: Literal["debug", "info", "warning"] = "warning"
+    secret: Optional[str] = None
+    private_key_path: Optional[pydantic.FilePath] = None
```

Comment:
Same comment here as `password` and `authentication` .. if we do not offer a private key authentication method, we do not need this option.

Reply:
removed the line as we Postgres does not offer key:pair authentication